README.md

```diff
@@ -226,6 +235,8 @@ The remaining properties
 `iconset` (defaults to `Octicons`),
 `data`, and
 `priority` (defaults `50`)
+`color`
+`background`
```

Comment:
I think this should be _"`data`, `priority` (defaults `50`), `color`, and `background`"_ could you update it accordingly?